### PR TITLE
Wiki Reorg: Videos

### LINKED ISSUE #4073: Update Video Tutorial Section
The video tutorial section needs to include all new tutorials (technical explainers). It might benefit from a fresh UI experience with a matrix layout highlighting the main topics of the videos (Accounts, Staking, Dashboard, Advanced, etc)

### PR DESCRIPTION
**Pull request by filippoweb3**
See [this issue](https://github.com/w3f/polkadot-wiki/issues/4376) and [this google doc](https://docs.google.com/document/d/1Fl8ReBIzcaQEOyF6s03d7MEcgk8aTVEt7xjmzAk_k5I/edit) for more information about the Wiki reorg.

### CODE REVIEW

**Comment by filippoweb3** [comment]
@DrW3RK @CrackTheCode016 I reviewed the video pages and proposed a more friendly version with links to the Youtube channel's playlists. let me know what you think!

File: docs/learn/learn-video-tutorials.md [hunk 1]
```diff
@@ -1,60 +1,160 @@
...
-- [Polkadot Webinars](https://www.youtube.com/watch?v=9B10wX9Mphc&list=PLOyWqupZ-WGsLEJq0sRyvUD-pWuVwA5yg)
-- [How to protect yourself from scams](https://www.youtube.com/watch?v=WqDlG4sy_i4)
+For more videos see all playlists on the
+[Polkadot YouTube channel](https://www.youtube.com/@PolkadotNetwork/playlists).
 
-## Tutorials
+:::
 
-### Accounts, Transfers & Staking
+## Trending Videos
```
Comment (CrackTheCode016) [inline docs/learn/learn-video-tutorials.md]:
```suggestion
## Ongoing Series

Learn about Polkadot and Substrate at a deeper level with these ongoing deep dives, technical seminars, and initiatives within the ecosystem.

```

**Review by CrackTheCode016 — CHANGES_REQUESTED** [review]
I love this layout!  The main thing for me is just the images, they seem pixelated.  Other than that, everything looks good to me

**Comment by filippoweb3** [comment]
@CrackTheCode016 the images are pixelated because those are thumbnails and I did my best :) I currently do not have the original but I asked marcomm ;) They are small anyway within the page but of course I asked to create some good ones so that we can add them in a future PR.

**Review by CrackTheCode016 — APPROVED** [review]

**Pull request merged**